src/tailwind.css

```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css?family=Libre+Franklin:300,400,600');
+@import url('https://fonts.googleapis.com/css?family=Libre+Franklin:300,400,600,700,800,900');
```

Comment:
Daaamn... do we really need all of those? This add a lot into the first loading time I will assume.